vmm/src/vm.rs

```diff
@@ -351,6 +355,7 @@ impl Vm {
             &exit_evt,
             &reset_evt,
             vmm_path,
+            &io_bus,
```

Comment:
This is fine for now but when you follow up with the change to make the bus hold `Weak`s can you try and move this back into `DeviceManager`

Reply:
Ok! noted ;)